src/components/home/KidHome.tsx

```diff
@@ -117,7 +118,7 @@ function KidHome() {
       <KidSummary />
       <WalkingDongilSection />
       <PendingDongilSection
-        onDeleteCheckOpen={openDeleteCheckSheet}
+        onWarningDeleteSheetOpen={openWarningDeleteSheet}
```

Comment:
```suggestion
        onWarnDeleteSheetOpen={openWarnDeleteSheet}
```
영문법상 동명사보다 동사가 맞는것 같습니다.

Reply:
`open<바텀시트컴포넌트이름><바텀시트타입이름>Sheet ` 로 통일하고 있습니다

Reply:
네 그건 저도 파악했는데, '삭제경고시트'를 영어로 옮기면 WaringDeleteSomeSheet가 영문법상 맞지 않고 WarnDeleteSomeSheet가 맞습니다.